molgenis/molgenis-emx2
# feat: Fetch last update column in single gql query

see https://preview-emx2-pr-4319.dev.molgenis.org/apps/central/#/ 

Replace n queries for n schema's with single fetch for all schema query to fetch lastUpdated table data

What are the main changes you did:
- Add query to fetch the last updated event for each schema in a single query
- Expose lastUpdate query via gql endpoint ( at db level, admin only)
- update central app fontend
  - replace fetch for each schema with
  - single call to lastUpdate query


how to test:
go to central app
- sign-in as non admin -> no last update data should be shown and no query should be fired 
- sign in as admin -> last update column should be shown, a single query should be fired to fetch the lastUpdate data, the schema's that have a changelog table show had a entry in the last update column showing the last change and a link to the details.

todo:
